src/openloco/platform/platform.posix.cpp

```diff
@@ -12,6 +12,7 @@
 
 int main(int argc, const char * * argv)
 {
+    openloco::lpCmdLine((char *)argv[0]);
```

Comment:
`lpCmdLine` is actually the full command line given to the application. You would need to join argv together with space separated, quotes if necessary. Or just leave it unset and implement the parser wherever it is.